spec/models/concerns/courses/edit_options/start_date_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe Courses::EditOptions::StartDateConcern do
+  let(:example_model) do
+    klass = Class.new do
+      include Courses::EditOptions::StartDateConcern
+
+      def provider; end
+    end
+
+    klass.new
+  end
+
+  let(:provider) { build(:provider) }
+
+  before do
+    allow(example_model).to receive(:provider).and_return(provider)
```

Comment:
Similar to my other comment - I feel an easier way to do this would be setting the attribute directly

Reply:
Done.